src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,98 @@
+package racingcar;
+
+import java.util.*;
+
+import static camp.nextstep.edu.missionutils.Randoms.pickNumberInRange;
+import static camp.nextstep.edu.missionutils.Console.readLine;
+
+public class Car{
+    Map<String,String> carInformation = new LinkedHashMap<>();
+    int repeatCount;
+    public Car() {
+        init();
+    }
+
+    public void init(){
+        System.out.println("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)");
+        String carName = readLine();
+        for (String name : carName.split(",")){
+            carInformation.put(name.trim(),"");
+        }
+        CheckException.checkRightCarName(carInformation.keySet());
+        repeatCount = User.howMuchTry();
+    }
+
+    private int createRandomNumber() {
+        return pickNumberInRange(0,9);
+    }
+
+    private boolean checkGoOrStop(int num){
+        if (num >= 4){
+            return true;
+        }
+        return false;
+    }
```

Comment:
```suggestion
    private boolean checkGoOrStop(int num){
        return num >= 4;
    }
```
이렇게 줄여볼 수 있을 거 같아요 😄